client/elements/text/sc-bilara-segmented-text.js

```diff
@@ -93,6 +92,12 @@ class SCBilaraSegmentedText extends SCLitTextPage {
     this.translationTextSelector = '.translation .text';
     this.rootTextSelector = '.root .text';
     this.commentSpanRectInfo = new Map();
+    
+    this._hashChangeHandler = () => {
+      setTimeout(() => {
+        this._scrollToSection(window.location.hash.substr(1));
+      }, 0);

```

Comment:
You can omit the second parameter it's the default value. Btw why you use setTimeout is there any reason for that?